tasks/chimp.js

```diff
@@ -20,8 +20,12 @@ module.exports = function(grunt) {
     // Make this task async
     var done = this.async();
     var options = this.options();
+    var chimpExecutable = '/node_modules/.bin/chimp';
+    if (process.platform === 'win32') {
```

Comment:
nope. that's fine by me as is. i just don't do enough windows to know. will merge and then do a patch release.